### ISSUE TITLE
[Bug]: If you add a person to a waiting list not in the profile, the cursor goes to the search field where you can start to type their name.  If you click in that field with your mouse you can no longer type anything in the box

### ISSUE THREAD

**Issue by ClaireMGitHub** [LamplightCRM/feedback#361]
### Do we know who you are?

MF Hackney Migrant Support

### What doesn't work?

If you add a person to a waiting list not in the profile, the cursor goes to the search field where you can start to type their name.  If you click in that field with your mouse you can no longer type anything in the box

### What did you see instead? Did you get a 'Problem saving' message? A 500 error (check Inspector)? Or something else?

None

### Can you reliably make the error happen repeatedly?

- [ ] Yes, every time I try
- [ ] Yes, most times I try
- [ ] No it seems random

### Please paste any error messages from the console that appear when this happens

16217
